docs/geedocs/docsrc/answer/geeServerAdmin/publishDefaultDatabase.rst

```diff
@@ -0,0 +1,41 @@
+|Google logo|
+
+========================
+Publish default database
+========================
+
+.. container::
+
+   .. container:: content
+
+      Google Earth Enterprise (GEE) Server allows users to publish one 3D
+      database as the default database. This is the database that will be
+      served when Earth Client signs into the base URL without including
+      a database name.
+
+      .. rubric:: Publishing as Default
+
+      To make a 3D database the default database, simply check the "Default
+      Database" checkbox when publishing the database.
+
+      |Publish as default|
+
+      This can also be done from the command line by including the ``--setecdefault``
+      option with the ``geserveradmin --publishdb`` command.
+      
+      e.g., ``geserveradmin --publishdb SFDb_3d --targetpath SFDb_3d --setecdefault``
+
+      Note that only one database can be the default, and the "last one wins" if
```

Comment:
Minor: again, this is picky, but I'm not sure I like the "last one wins" phrasing. Maybe something like "If you publish a default database, the previous default will be cleared."